<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ComicDesktop.qrc">
    <normaloff>:/ComicDesktop/Resources/XylasoftComicsLogoSmall.png</normaloff>:/ComicDesktop/Resources/XylasoftComicsLogoSmall.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>396</width>
        <height>469</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBoxGeneral">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Reopen Comics on Restart :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBoxReopenSession">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Show Full File Path :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBoxShowPath">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxViewing">
         <property name="title">
          <string>Viewing</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Default page view : </string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Default view state : </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxImageState">
              <item>
               <property name="text">
                <string>Scaled</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fit to Width</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fit to Height</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fit to Width and Height</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxImageView">
              <item>
               <property name="text">
                <string>1 Page</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2 Pages</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Background : </string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="ColorPicker" name="widgetColor" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Thumbnail Size (pixels) : </string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="spinBoxThumbnailSize">
              <property name="minimum">
               <number>75</number>
              </property>
              <property name="maximum">
               <number>600</number>
              </property>
              <property name="value">
               <number>120</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Scaled Zoom :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="sliderScaledZoom">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>4</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Allow magnification :</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QCheckBox" name="checkBoxAllowMagnification">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Page Numbering</string>
            </property>
            <layout class="QFormLayout" name="formLayout_4">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Show :</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="chkShowPageNumber">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Location :</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="cboPageLocation">
               <item>
                <property name="text">
                 <string>Top</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Bottom</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Color :</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="ColorPicker" name="widgetPageColor" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Two-page viewing</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Pages to move : </string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBoxTwoPageMove">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>2</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Display wide pages as single :</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="checkBoxAutodetectWide">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonRestoreDefaults">
       <property name="text">
        <string>Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QWidget</extends>
   <header>ColorPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxImageView</tabstop>
  <tabstop>comboBoxImageState</tabstop>
  <tabstop>spinBoxTwoPageMove</tabstop>
 </tabstops>
 <resources>
  <include location="ComicDesktop.qrc"/>
 </resources>
 <connections/>
</ui>
